expressjs/src/routes/events/endpoint.js

```diff
@@ -0,0 +1,99 @@
+const { HTTP } = require('cloudevents')
+const openapi = require('../../lib/openapi')
+const EventStore = require('./store')
+const devdata = require('./devdata')
+
+const store = new EventStore()
+devdata.forEach(event => store.add(event))
+
+module.exports = async app => {
+  app.get('/events', streamDoc, (_, res) => {
+    res.set('Content-Type', 'text/event-stream')
+    res.set('Cache-Control', 'no-cache')
+    res.set('Connection', 'keep-alive')
+    res.set('X-SSE-Content-Type', 'application/cloudevents+json')
+    res.set('transfer-encoding', 'chunked')
+    res.flushHeaders()
+
+    const stream = store.createStream(res)
+    stream.stream()
+  })
+
+  app.post('/events', eventDoc, (req, res) => {
+    try {
+      const ce = HTTP.toEvent({ headers: req.headers, body: req.body })
+      ce.validate()
+      store.add(ce)
+      res.status(201).end()
```

Comment:
We should log the event in the same way [the Knative's event-display image](https://knative.dev/docs/eventing/flows/sequence/sequence-reply-to-event-display/#inspecting-the-results) does it:

```
☁️  cloudevents.Event
Validation: valid
Context Attributes,
  specversion: 1.0
  type: samples.http.mode3
  source: /apis/v1/namespaces/default/pingsources/ping-source
  id: e8fa7906-ab62-4e61-9c13-a9406e2130a9
  time: 2020-03-02T20:52:00.0004957Z
  datacontenttype: application/json
Data,
  {
    "id": 0,
    "message": "Hello world"
  }
```

Reply:
From my point of view, this could be done in a follow up PR.

Reply:
Tracked in https://github.com/openshift-knative/showcase/issues/15